First pass at L1Trigger Layer1 DQM module

nsmith- opened the pull request:
For integration into production online DQM sequence

@apana, @bortigno, you wanted to be informed about this

I forward-ported this to 81X as well, #13734

nsmith- committed:
First pass at L1Trigger Layer1 DQM module

For integration into production DQM sequence

cmsbuild commented:
A new Pull Request was created by  @nsmith- (Nicholas Smith) for CMSSW_8_0_X.

It involves the following packages:

DQM/Integration
DQM/L1TMonitor


@cmsbuild, @vanbesien, @deguio, @davidlange6 can you please review it and eventually sign? Thanks.
@threus, @batinkov this is something you requested to watch as well.
@slava77, @degano, @smuzaffar you are the release manager for this.

cms-bot commands are list here cms-sw/cmssw#13028

nsmith- committed:
CaloLayer1 DQM bugfix

TPs start at 1!

nsmith- committed:
Calo Layer1 update: remove separate received HF digi

HF and HCAL digis will be combined in L1CaloLayer1 unpacker in this:
https://github.com/cms-sw/cmssw/pull/13723
This branch would then cause an error if we include HF digis as
currently implemented.

nsmith- commented:
Superceded by #13763
